src/App.css

```diff
@@ -1,38 +1,62 @@
-.App {
-  text-align: center;
+.itemNewsRoom {
```

Comment:
I don't think we should need this flex type stuff. What is this doing? Controlling the content of the card?

Reply:
flex is ordering the card content

Reply:
@tmccoy529 once i made it down i noticed, that, I shared a link to some guidance on that.